app/views/bills/index.html.erb

```diff
@@ -0,0 +1,10 @@
+<p style="color: green"><%= notice %></p>
+<h1>Bills</h1>
+<div id="bills">
+  <% @bills.each do |bill| %>
+    <%= render bill %>
+    <p>
+      <%= link_to "Show this bill", group_bill_path(bill.group_id, bill)%>
+    </p>
+  <% end %>
+</div>
```

Comment:
- Hello @musangamfure, you have done well so far, but according to the previous reviewer, please style the transactions page as it is in the design template. From what the previous reviewer marked in the screenshot, the marked section, i.e the name of the category and the total amount, should be directly under the blue navbar. Please fix this